components/layout/Nav.js

```diff
@@ -59,10 +80,13 @@ const StyledNav = styled.nav`
     background: none;
     border: 0;
     cursor: pointer;
-    @media (max-width: 700px) {
-      font-size: 10px;
-      padding: 0 10px;
+    @media (max-width: 750px) {
+      padding: 0 1rem;
+      letter-spacing: 0.1rem;
     }
+  }
+  a.loggedOut,
+  b.loggedOut {
```

Comment:
What's `b`?
Also; as you're already using class names why do you also need to describe the html element? 

Anyways; you should change it to styled component

Reply:
I removed that right after the pull request, you're very right! Thank you!